dclab/definitions/meta_const.py

```diff
@@ -124,6 +125,25 @@
         ["target duration", float, "Target measurement duration [min]"],
         ["target event count", fint, "Target event count for online gating"],
     ],
+    # All qpi-related keywords
+    "qpi": [
+        # experiment-related qpi metadata
+        ["wavelength", float, "Laser wavelength [nm]"],
+        ["medium index", float, "Refractive index of Medium"],
+        # post-analysis-related qpi metadata
+        ["software version", str, "Software version used"],
+        # calculation of pha and amp from hologram
+        ["filter_name", str, "Fourier filter used"],
+        ["filter_size", str, "Fourier filter size [1/pix]"],
+        ["filter_size_interpretation", str, "How to interpret filter size"],
+        ["sideband_freq", f2dfloatarray,
+         "Frequency coordinates of the sideband [1/pix]"],
+        # processing of hologram
+        ["padding", fbool, "If padding was used"],
+        ["fit_offset", str, "Fitting offset for processing hologram"],
+        ["fit_profile", str, "Fitting profile for processing hologram"],
+        ["border_px", fint, "Size of border for processing hologram [pix]"],
+    ],
```

Comment:
A was thinking of something more along the lines of this: https://github.com/RI-imaging/qpretrieve/blob/4446d0d14b8dbe74c37f0d81d24e214826c9f2a8/qpretrieve/interfere/if_oah.py#L9-L15

and padding/subtract mean from

https://github.com/RI-imaging/qpretrieve/blob/4446d0d14b8dbe74c37f0d81d24e214826c9f2a8/qpretrieve/interfere/base.py#LL18C30-L18C59

In the end, hologram processing will be done by qpretrieve, correct?

Reply:
Not sure what you "mean" ;). Do you mean the QPI-related processing config info should be in a dictionary-like structure, or?

> In the end, hologram processing will be done by qpretrieve, correct?

Yes! But because I've been  actually have been using it in the background